src/dbnode/namespace/convert_test.go

```diff
@@ -76,6 +76,7 @@ var (
 			RetentionOptions:      &validRetentionOpts,
 			SchemaOptions:         testSchemaOptions,
 			ExtendedOptions:       validExtendedOpts,
+			StagingState:          &nsproto.StagingState{Status: nsproto.Status_INITIALIZING},
```

Comment:
Should there be some change to expected data to mirror/verify this?

nit; probably worth a test for conversion the other way

Reply:
Ahh nice catch. Missed updating the expectation block